AI-Upscaling-With-NPU/tools/install_pixi.ps1

```diff
@@ -0,0 +1,169 @@
+<#
+.SYNOPSIS
+    Pixi install script.
+.DESCRIPTION
+    This script is used to install Pixi on Windows from the command line.
+.PARAMETER PixiVersion
+    Specifies the version of Pixi to install.
+    The default value is 'latest'. You can also specify it by setting the
+    environment variable 'PIXI_VERSION'.
+.PARAMETER PixiHome
+    Specifies Pixi's home directory.
+    The default value is '$Env:USERPROFILE\.pixi'. You can also specify it by
+    setting the environment variable 'PIXI_HOME'.
+.PARAMETER NoPathUpdate
+    If specified, the script will not update the PATH environment variable.
+.LINK
+    https://pixi.sh
+.LINK
+    https://github.com/prefix-dev/pixi
+.NOTES
+    Version: v0.34.0
+#>
+param (
+    [string] $PixiVersion = 'latest',
```

Comment:
If we want to use just the particular verified version of PIXI I wonder if leaving the default 'latest' option is safe. I mean if anyone will make changes to the global variable that specifies the PIXI_VERSION  or delete it (for some reasons) we will be installing the latest version of PIXI which was not verified and checked. I wonder if it wouldn't be better to throw an error when version isn't specified instead of using the default 'latest' version.

Reply:
This has been changed to throw an error.